### ISSUE TITLE
Final Proofreading: Acknowledgments

### ISSUE THREAD

**Issue by samuel-lucas6** [cfrg/draft-irtf-cfrg-aegis-aead#80]
### Manual Review
https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#appendix-B-1
`AEGIS authenticated encryption algorithm` => There are multiple algorithms, although their CAESAR submission writes it this way.

https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#appendix-B-2
`the Pelican MAC` => MAC function? Not necessary though, and this is the name, just feels a bit weird to me.

https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#appendix-B-4.1.1
`vector and Daniel Bleichenbacher` => `, and Daniel`

https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#appendix-B-4.2.1
Remove the comma.

https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#appendix-B-4.3.1
`as well as Takanori Isobe and Mostafizar Rahman for` => `, as well as ... Rahman, for`

`commitment security` => `committing security`

https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#appendix-B-4.4.1
`Crypto Panel` => `Crypto Review Panel`

Merge with the bullet below? Doesn't really matter.

### Grammarly
Grammarly agrees with these grammatical changes.

**Comment by samuel-lucas6** [comment]
Just been looking at the test vectors, and didn't Soatok provide us with feedback and inspire the 'verification error' test vectors? He hasn't been acknowledged at the moment, and I don't see an issue with his nickname since we've mentioned Yawning Angel.

**Comment by jedisct1** [comment]
Indeed, we should definitely mention Soatok. Also for c7605eccee4d321f3deb14eef4d30bab2a27a3af

Done.
